go.mod

```diff
@@ -9,11 +9,19 @@ require (
 )
 
 require (
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+)
+
+require (
+	github.com/aws/aws-sdk-go v1.45.25
 	github.com/go-logr/logr v1.2.4 // indirect
 	github.com/go-task/slim-sprig v0.0.0-20230315185526-52ccab3ef572 // indirect
 	github.com/google/go-cmp v0.5.9 // indirect
 	github.com/google/pprof v0.0.0-20210407192527-94a9f03dee38 // indirect
+	github.com/openshift/rosa v1.2.27
 	github.com/stretchr/testify v1.7.0 // indirect
+	github.com/zgalor/weberr v0.8.2
```

Comment:
I dont think we need weberr in common, as its not used in ROSA. 

We can just return a generic error, and leave CS wrap in weberr as needed.